packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -431,7 +431,7 @@ function OrgSelect({
               <MenuSeparator />
               <MenuItem
                 onKeyDown={onKeyDown}
-                href={urls.viewAllOrganizations}
+                href={urls.viewAllOrganizations!}
```

Comment:
Weren't the typing changes in mongo nav explicitly to address the typing issue here?

Reply:
👍